pytest_celery/test_services/message_brokers/__init__.py

```diff
@@ -1,6 +1,7 @@
 from __future__ import annotations
 
 from pytest_celery.test_services.message_brokers.base import MessageBroker
+from pytest_celery.test_services.message_brokers.rabbitmq import RabbitMQBroker
 from pytest_celery.test_services.message_brokers.redis import RedisBroker
 
-__all__ = ["MessageBroker", "RedisBroker"]
+__all__ = ["MessageBroker", "RedisBroker", "RabbitMQBroker"]
```

Comment:
Since this is a constant and it's the main entry point of the API, using a tuple is a better idea.

```suggestion
__all__ = ("MessageBroker", "RedisBroker", "RabbitMQBroker")
```